client/src/pages/organizations/Approvals.js

```diff
@@ -23,7 +23,9 @@ export default class OrganizationApprovals extends Component {
 						<tbody>
 							{step.approvers.map((position, approverIdx) =>
 								<tr key={position.uuid} id={`step_${idx}_approver_${approverIdx}`} >
-									<td>{position.person && <LinkTo person={position.person} />}</td>
+									<td>{(position.person && position.person.uuid)
+										? <td>{position.person && <LinkTo person={position.person} />}</td>
```

Comment:
`? <td><LinkTo person={position.person} /></td>` is sufficient since the condition is already checked.

Reply:
This condition is also used to display positions in Laydown.js, and thus I though there may be an edge case which does not trigger this line if missing the second part. I will change this, and suggest to also change the other condition in the future.

Reply:
You made the wrong change… You should have changed the exact line I commented on to the exact line I suggested. You changed the line *above* the line I commented on.

Reply:
Oops, you are right :)

Reply:
I reverted my first change and removed the unncessary check.